.eslintignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+dist
+test
+lib
```

Comment:
Why ignore `test` and `lib` folder?

Reply:
test is a complete rewrite in ts and lib will be deleted